src/lib/api/feed-manip.ts

```diff
@@ -205,8 +213,22 @@ export class FeedTuner {
       dryRun: false,
     },
   ): FeedViewPostsSlice[] {
+    const replyRefMap: ReplyRefMap = new Map()
+    for (const item of feed) {
+      for (const ancestor of [item.reply?.parent, item.reply?.root]) {
+        // trust server for blocked and not found reply parent/root refs
+        if (
+          AppBskyFeedDefs.isPostView(ancestor) ||
+          AppBskyFeedDefs.isBlockedPost(ancestor) ||
+          AppBskyFeedDefs.isNotFoundPost(ancestor)
+        ) {
+          replyRefMap.set(ancestor.uri, ancestor)
```

Comment:
i wonder if we can make this more scoped to clarify its purpose. e.g. `blockedPostUris` and `notFoundPostUris` sets get threaded through and then we check against those. we never really need the actual content of the post so this feels like it's exposing too much of the "not really consistent" picture to the slice logic. i'd like to keep it more limited.